packages/client/src/Components/Mapping/index.jsx

```diff
@@ -8,6 +9,7 @@ import markerFor from '../../Helpers/markerFor'
 import '../../Helpers/mousePosition'
 
 import './styles.scss'
+import './leaflet.zoomhome.js'
```

Comment:
Can you add a `//TODO: Refactor` above this line. It's fine for now but I don't want us to get into the habit of doing naked imports on javascript. It's not worth doing for the Feb demo as it requires converting `zoomhome` into an actual model but it should definitely be done for the rebuild.